sql/hive/src/test/scala/org/apache/spark/sql/hive/CachedTableSuite.scala

```diff
@@ -203,4 +203,18 @@ class CachedTableSuite extends QueryTest {
     sql("DROP TABLE refreshTable")
     Utils.deleteRecursively(tempPath)
   }
+
+  test("SPARK-10327 Cache Table is not working while subquery has alias in its project list") {
```

Comment:
In sql/core, the existed data like `testData` stuff are stored in memory, the physical plan always be `InMemoryColumnarTableScan`, then it is hard to tell if it's cached.